<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>confirmar_anio</class>
 <widget class="QDialog" name="confirmar_anio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>265</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>265</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>265</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f6fa;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetPrincipal" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: #F7F9FC;
color: #2d2d2d;
}
QLineEdit {
    border: 2px solid #2980b9;
    border-radius: 12px;
    padding: 3px 8px;
    background-color: white;
	color: #2d2d2d;
}</string>
     </property>
     <widget class="QLabel" name="lblTitulo_login">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>325</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #2c3e50; 
}</string>
      </property>
      <property name="text">
       <string>Seleccione el nuevo a√±o escolar</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAperturar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::ClickFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #2980b9;
    color: #FFFFFF;
    border: none;
    padding: 8px 10px;
    border-radius: 12px;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
      </property>
      <property name="text">
       <string>Aperturar</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/resources_ui.qrc">
        <normaloff>:/icons/confirm_white.png</normaloff>:/icons/confirm_white.png</iconset>
      </property>
     </widget>
     <widget class="QFrame" name="frameAnio_nuevo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>230</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: white;
	border: 1.5px solid #2980b9;
	border-radius: 12px;
}
QComboBox QAbstractItemView {
    border: 1px solid #ccc;
    border-radius: 5px;
    background-color: white;
    color: #333;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QComboBox" name="cbxAnio_nuevo">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>5</y>
         <width>210</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: transparent;</string>
       </property>
       <property name="iconSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnCancelar">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>141</width>
        <height>35</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::ClickFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #e74c3c;
    color: #FFFFFF;
    border: none;
    padding: 8px 10px;
    border-radius: 12px;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/resources_ui.qrc">
        <normaloff>:/icons/cancelar_w2.png</normaloff>:/icons/cancelar_w2.png</iconset>
      </property>
     </widget>
     <zorder>btnAperturar</zorder>
     <zorder>lblTitulo_login</zorder>
     <zorder>frameAnio_nuevo</zorder>
     <zorder>btnCancelar</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/resources_ui.qrc"/>
  <include location="../icons/resources_ui.qrc"/>
 </resources>
 <connections/>
</ui>
